devicetypes/stelpro/stelpro-ki-thermostat.src/stelpro-ki-thermostat.groovy

```diff
@@ -0,0 +1,541 @@
+/*
+ *  Copyright 2017 Stelpro
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Stelpro Ki Thermostat
+ *
+ *  Author: Stelpro
+ *
+ *  Date: 2018-04-04
+ */
+
+preferences {
```

Comment:
can you move this before the metadata

Reply:
i am sorry. i meant after the metadata. the metadata is the first code block in most of code base